keras_nlp/layers/preprocessing/mlm_mask_generator.py

```diff
@@ -83,6 +83,45 @@ class MLMMaskGenerator(keras.layers.Layer):
             vocabulary_size=10, mask_selection_rate=0.5, mask_token_id=0, \
             mask_selection_length=5)
     >>> masker(tf.ragged.constant([[1, 2], [1, 2, 3, 4]]))
+
+    With TensorFlow Dataset
+    >>> # Load dataset
```

Comment:
I'd suggest a fenced code block here (triple backticks), otherwise you need to begin every line with either >>> (new line) or ... (continuation line).